connectivity/suite.go

```diff
@@ -16,18 +16,90 @@ package connectivity
 
 import (
 	"context"
+	_ "embed"
 
 	"github.com/cilium/cilium-cli/connectivity/check"
 	"github.com/cilium/cilium-cli/connectivity/tests"
 )
 
+var (
+	//go:embed manifests/allow-all.yaml
+	allowAllPolicyYAML string
+
+	//go:embed manifests/client-egress-only-dns.yaml
+	clientEgressOnlyDNSPolicyYAML string
+
+	//go:embed manifests/client-egress-to-echo.yaml
+	clientEgressToEchoPolicyYAML string
+
+	//go:embed manifests/client-ingress-from-client2.yaml
+	clientIngressFromClient2PolicyYAML string
+
+	//go:embed manifests/client-egress-toFQDNs-google.yaml
```

Comment:
Let's stick to lowercase here so we can match the filename with the object name (since there's only 1 policy per file right now):
```suggestion
	//go:embed manifests/client-egress-to-fqdns-google.yaml
```

Reply:
Fixed.